inference-engine/src/mkldnn_plugin/mkldnn_memory.h

```diff
@@ -94,6 +117,12 @@ class MKLDNNMemory {
 public:
```

Comment:
Same comment. We need to clean up Memory API.

Reply:
As it was discussed, the API cleanup is planned in a separate PR.